control-plane/api/v1alpha1/servicedefaults_types_test.go

```diff
@@ -193,7 +193,7 @@ func TestServiceDefaults_ToConsul(t *testing.T) {
 						PassiveHealthCheck: &capi.PassiveHealthCheck{
 							Interval:                2 * time.Second,
 							MaxFailures:             uint32(20),
-							EnforcingConsecutive5xx: uint32(100),
+							EnforcingConsecutive5xx: uint32Pointer(100),
```

Comment:
We recently went through and moved everything in the `consul-k8s` repo to use the pointer `k8s.io/utils/pointer` library for these, could you swap them out for it please? 
```suggestion
							EnforcingConsecutive5xx: pointer.Uint32(100),
```

Reply:
Great to know, thanks!